R/case.R

```diff
@@ -1,37 +1,60 @@
 #' Create a validation specification, test case, or test code file
 #'
-#' @param case_name The name/path of the test case. Cases can be named with your
+#' @param name The name/path of the test case. Cases can be named with your
 #'   file system separator and will be organized as a directory structure. Cases
-#'   are located at `./inst/validation/cases/{case_name}`.
+#'   are located at `./inst/validation/test_case/{name}`.
+#' @param username the username to insert into the specification as the author.
+#' @param open should the newly make specification open for editing.
 #' @param pkg Top-level of package
 #'
 #' @export
 #'
 #' @rdname new_item
 #'
 #' @examples
+#' \dontrun{
+#' # TODO: error during devtools::check
 #' package_dir <- tempdir()
 #'
 #' vt_use_validation(package_dir)
-#'
 #' # Create spec at the cases top level `inst/validation/cases/spec1`
-#' vt_use_case("spec1", package_dir)
+#' vt_use_test_case("spec1", package_dir)
 #' # Create spec at `inst/validation/cases/regTests/Update2/spec2`
-#' vt_use_case("regTests/Update2/spec2", package_dir)
-vt_use_case <- function(case_name, case_content, pkg = ".") {
-
-  case_path <- create_item(pkg, "cases", case_name)
-
-  # Create the content to write
-  content <- paste0(c(
-    "#' @section Last Updated By:",
-    paste0("#' ", Sys.getenv("USER")),
-    "#' @section Last Update Date",
-    paste0("#' ", as.character(Sys.Date())),
-    paste0("+ ", case_content, collapse = ""),
-    collapse = "",
-    sep = "\n"
-  ))
-
-  writeLines(content, con = case_path)
+#' vt_use_test_case("regTests/Update2/spec2", package_dir)
+#' }
+#' @importFrom usethis edit_file
+vt_use_test_case <- function(name, username = vt_username(), open = interactive(), pkg = ".") {
+
+  name <- vt_set_ext(name, ext = "md")
+
+  case_name <- create_item(pkg, "test_case", name)
+
+  ## if the file didnt exist before, populate with contents
+  if (file.size(case_name) == 0){
+
+    # Create the content to write
+    content <- paste0(c(
+      "#' @section Last Updated By:",
```

Comment:
same as for `vt_use_spec`:

- will need to update due to #16 
- missing @title field